### PR TITLE
MVP-423: Added basic configs

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[MVP-423](https://citz-imb.atlassian.net/jira/browse/MVP-423)

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@bcgov/citz-imb-sso-js-core",
+  "version": "0.0.0",
+  "description": "BCGov JavaScript SSO integration Core",
+  "author": "CITZ IMB Common Code <citz.codemvp@gov.bc.ca>",
+  "license": "Apache-2.0",
+  "main": "build/index.js",
+  "types": "build/bundle.d.ts",
+  "scripts": {
+    "build": "npm run clean:prebuild && rollup -c rollup.config.mjs && npm run build:dts && npm run clean:postbuild",
```
Comment (TaylorFries) [inline package.json:10]: hit this error when trying to run build script: 
```
src/index.ts → build...
[!] RollupError: Could not resolve entry module "src/index.ts".
    at getRollupError (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/parseAst.js:282:41)
    at Object.error (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/parseAst.js:278:42)
    at ModuleLoader.loadEntryModule (/Users/tafriese/Documents/code/citz-imb-sso-js-core/node_modules/rollup/dist/shared/rollup.js:20183:32)
    at async Promise.all (index 0)
```

Comment (BradyMitch) [reply package.json:10]: Yeah it wont build yet until theres src files

**Review by TaylorFries — APPROVED** [review]
all changes follow standards and no issues detected.

**Comment by adamkroonbcgov** [comment]
fails all zero of the tests. looks good.

**Pull request merged**